calblueprint/siren
# Fonts + Colors

# ✨ New in this PR ✨

## One liner: what did you do? 🚨 

## Coverage 🙆‍♀️
_use this section to break up your task into submodules and track progress_ 
 - [X] loading fonts logic
 - [x] upload remaining fonts 
 - [x] upload colors 

## How can the reviewer test your code? 👩‍💻
Go to HomeScreen + verify that fonts work using `TextBold` and `TextRegular` !!! 

## Any bugs you encountered or still having trouble with? 🐛
Packages are weird. Here's a log of packages we settled on + commands. VERSIONS MATTER! 
- expo-font: `expo install expo-font`
- expo-app-loading: `expo install expo-app-loading `
- styled-components/native:` npm i styled-components`
- @types/styled-components: `npm i -D @types/styled-components@5.1.6`

## Resources 📔
[Loading fonts in Expo in a functional component ](https://docs.expo.dev/guides/using-custom-fonts/)
[What does AppLoading do? ](https://docs.expo.dev/versions/latest/sdk/app-loading/)

🧜‍♀️ cc: @shannonbonet @phoebeli23 @oahnh
